### PR TITLE
Added Fast Style Transfer showcase

### PR DESCRIPTION
**Pull request by jphong89**
- Added style transfer net & template
- Added mscoco dataset api

### CODE REVIEW

File: fastestimator/architecture/stnet.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
+import tensorflow as tf
+
+class ReflectionPadding2D(tf.keras.layers.Layer):
```
Comment (TortoiseHam) [inline fastestimator/architecture/stnet.py]: is this class generic enough that it would be useful to have as a separate file?

Comment (jphong89) [reply fastestimator/architecture/stnet.py]: I think this type of padding is specific to problems like image to image translation (e.g.,  cycleGAN) and style transfer.
